package.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
it could be me but I think the formatting is messed up on this file.

Reply:
It is formatting.